docs/index.rst

```diff
@@ -2,10 +2,61 @@ Welcome to "djangocms-versioning"'s documentation!
 ==================================================
 
 .. toctree::
-   :maxdepth: 3
-   :caption: Contents:
+   :maxdepth: 2
+   :caption: Quick Start:
 
    versioning_integration
+
+.. toctree::
+   :maxdepth: 2
+   :caption: API Reference:
+
+   advanced_configuration
    signals
    customizing_version_list
 
+.. toctree::
+   :maxdepth: 2
+   :caption: Internals:
+
+   admin_architecture
+
+
+Glossary
+--------
+
+.. glossary::
+
+    version model
+        A model that stores information such as state (draft, published etc.),
+        author, created and modified dates etc. about each version.
+
+    content model
+        A model with a one2one relationship with the
```

Comment:
I suggest add in the image that I put int the `docs/_static` folder which illustrates this.

Reply:
See here: https://divio-ch.atlassian.net/wiki/spaces/FIL/pages/668041217/Gaps+Documentation#GapsDocumentation-djangocms-versioning(Divio)

Kunjal found this very helpful. Not sure if Glossary is the right place for it.

Reply:
Can use https://www.sphinx-doc.org/en/master/usage/extensions/graphviz.html for that

Reply:
I'll have a go at adding a graphviz version of that diagram to the versioning integration page as that's really where I try to explain these concepts in more detail.